components/Room.tsx

```diff
@@ -27,7 +27,6 @@ function Room({
 }) {
   const [isParticipantsListVisible, setIsParticipantsListVisible] =
     useState<boolean>(false);
-  const [audioOutputDeviceId, setAudioOutputDeviceId] = useState<string>();
```

Comment:
I used the opportunity to do a small refactor.

We don't need this state here as this value is already part of `TelnyxMeetContext`.